draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -315,6 +314,25 @@ Any claims used that are not understood MUST be ignored.
 Additional validation rules MAY apply, but their use is out of the scope of this
 specification.
 
+## Public Key for Issuer-signed JWTs Verification {#public-key-for-issuer-signed-jwts-verification}
+
+A recipient of the SD-JWT VC MUST apply the following rules to verify the
+Issuer-signed JWT corresponds to the Issuer:
```

Comment:
```suggestion
A recipient of the SD-JWT VC MUST use the first applying strategy of the following list to verify the
Issuer-signed JWT corresponds to the Issuer:
```

Reply:
Why is first applying needed. the options are mutually exclusive, aren't they?